applications/protocols_serialization/README.rst

```diff
@@ -0,0 +1,256 @@
+.. _protocols_serialization_server_app:
+
+Protocols serialization application
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The protocols serialization server runs full Bluetooth LE and OpenThread stacks and it exposes selected functions from these stacks over the UART.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+Protocols serialization application performs server part of the OpenThread and Bluetooth LE API calls serialization between two devices
+that communicate with each other using the UART interface. The application uses nRF RPC library and CBOR encoding to serialize function calls.
+
+Building and flashing application
+*********************************
+
+Complete the following steps to build and flash the server application into another nRF52840 DK:
+
+#. Open a terminal and navigate to the nRF Connect SDK installation directory.
+
+#. Navigate to the server application's directory:
+
+   .. code-block:: console
+
+      ~/ncs/nrf$ cd applications/protocols_serialization
+
+
+#. Build the server application:
+
+   .. code-block:: console
+
+      ~/ncs/nrf/applications/protocols_serialization$ west build -b nrf52840dk/nrf52840 -S "openthread;ble;debug"
+
+
+   Note that you can modify the list of enabled features, which by default includes **ble** support as well as **debug** logs.
+
+#. Flash the server application into an nRF52840 DK:
+
+   .. code-block:: console
+
+      ~/ncs/nrf/applications/protocols_serialization$ west flash --erase
+
+Testing
+*******
+
+For testing the application, use the protocol serialization client sample :ref:`nrf_rpc_protocols_serialization_client`.
+
+.. _protocols_serialization_server_app_connection:
+
+Connecting client and server
+============================
+
+The nRF52840 system-on-chip includes two UART peripherals: `uart0` and `uart1`. In the protocols serialization application, the `uart0` peripheral is used for shell and logging purposes, similarly to other applications and samples, while the `uart1` peripheral is used for sending OpenThread and Bluetooth LE remote procedure calls (RPCs).
```

Comment:
Consider using `.. tabs` to have descriptions regarding nRF52 and nRF54 on separate tabs.

Reply:
I didn't get this. Could you clarify or provide sample?

Reply:
Look for `.. tabs::` in the documentation :). Basically, if there are alternative paths for different platforms it's nice to have the instruction separated to have these paths described on separate tabs, so that a user doesn't have to scroll till the relevant part.